src/App.css

```diff
@@ -1,23 +1,223 @@
-.App {
-  text-align: center;
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+body {
+  font-family: 'Montserrat', sans-serif;
+  background-color: #fafafa;
 }
 
-.App-header {
-  background-color: #282c34;
+/* Header styles */
+
+header {
+  background-color: white;
+  border-bottom: 1px solid #e8e8e8;
   display: flex;
-  flex-direction: column;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  justify-content: space-between;
+  margin-bottom: 2.375rem;
 }
 
-nav {
+.title-nav {
+  width: 45vw;
   display: flex;
+  align-items: center;
+  margin: 0 0 0 7vw;
+}
+
+.title {
+  white-space: nowrap;
+  color: #0290ff;
+  font-size: 1.8rem;
+  width: 15rem;
+  font-weight: bold;
+  padding: 1rem 0;
+}
+
+nav {
+  margin-left: 0.7rem;
+}
+
+.login {
+  justify-content: flex-end;
+  margin-right: 7vw;
 }
 
 .link {
-  color: #61dafb;
   text-decoration: none;
+  text-transform: uppercase;
+  font-size: 0.83rem;
+  color: #121212;
+  margin-left: 1rem;
   padding: 1rem;
 }
+
+/* Book styles */
+
+.book {
+  height: 6.625rem;
```

Comment:
- [ ] I suggest to remove the height here too because it is just raising a little the height but in this case you used padding so if you want to make it a little more big you can raise the padding and it will look as you want. It is a better practice to not use properties for the same purpose. Keep coding 💪